.github/workflows/help-in-readme.yml

```
@@ -5,9 +5,9 @@ jobs:
   help-in-readme:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
       - name: Install uv
-        uses: astral-sh/setup-uv@v4
+        uses: astral-sh/setup-uv@v6
```

Seems there's a
```suggestion
        uses: astral-sh/setup-uv@v7
```
already. Same in a few other workflow files.